.ci/README.md

```diff
@@ -1,69 +1,62 @@
-2. Immediately after your downstream builder job is submitted, prevent submissions to the magic-modules repository.
-3. Once the downstream-builder container is regenerated (about 15 minutes) re-enable submissions to magic-modules and press retry on your downstream builder job.
-
-Disabling submissions to magic-modules can be done through the Admin console in GitHub.
-
-It is safe to have more than one downstream-push running at the same time due to this property, in the event of overruns.  Each run will either
-a) make no changes to any downstream and fail
-or
-b) atomically update every downstream to a fast-forward state that represents the appropriate HEAD as of the beginning of the run
-
-#### Something went wrong!
-Don't panic - this is all quite safe and we have fixed it before.  We store the state of the pusher tasks in tags (the "sync tags"), one per downstream, and it will be easy to get the system back up and running.  You can send a message to Nathan if you are anxious about running through these steps.  :)
-
-It's possible for a job to be cancelled or fail in the middle of pushing downstreams in a transient way.  The sorts of failures that happen at scale - lightning strikes a datacenter (ours or GitHub's!) or some other unlikely misfortune happens.  This has a chance to cause a hiccup in the downstream history, but isn't dangerous.  If that happens, the sync tags may need to be manually updated to sit at the same commit, just before the commit which needs to be generated, or some failed tasks might need to be run by hand.
-
-First, check their state: `git fetch origin && git rev-parse origin/tpg-sync origin/tpgb-sync origin/tf-oics-sync origin/tgc-sync` will list the commits for each of the sync tags. (If you have changed the name of the `GoogleCloudPlatform/magic-modules` remote from `origin`, substitute that name instead, such as `git fetch upstream && git rev-parse upstream/tpg-sync upstream/tpgb-sync upstream/tf-oics-sync upstream/tgc-sync`)
-
-In normal, steady-state operation, these tags will all be identical.  When a failure occurs, some of them may be one commit ahead of the others.  It is rare for any of them to be 2 or more commits ahead of any other.  If some of them are one commit ahead of the others, and there is no pusher task currently running, this means you need to reset them by hand and rerun the failed jobs.  If they diverge by more than one commit, or a pusher task is currently running, you will need to manually run missing tasks.
+```
+`git fetch origin && git rev-parse origin/tpg-sync origin/tpgb-sync origin/tf-oics-sync origin/tgc-sync`
+```
 
-### Divergence by zero commits
+### Transient GitHub failures
+Most downstream build failures are transient GitHub failures. To resolve these, click the "Retry" button in Cloud Build. This is safe because downstream builds are idempotent; if a commit has already been generated, we will not make a new commit.
```

Comment:
> Most downstream build failures are transient GitHub failures.

Do 5XX errors encapsulate most of these?

Reply:
I think they can also be 4XX (for example auth errors); I'm not sure if one or the other is more common.